### PR TITLE
[WFLY-15559] added webservices to source-transform

### PR DESCRIPTION
**Pull request by rsearls**
https://issues.redhat.com/browse/WFLY-15559 Move WildFly Preview 
      to a native Jakarta namespace variant of the webservices subsystem module

### CODE REVIEW

File: ee-9/source-transform/webservices/server-integration/pom.xml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+       <dependency>
+          <groupId>jakarta.ejb</groupId>
+          <artifactId>jakarta.ejb-api</artifactId>
+       </dependency>
+
+       <dependency>
+          <groupId>jakarta.xml.ws</groupId>
+          <artifactId>jakarta.xml.ws-api</artifactId>
+          <version>${jakarta.xml.ws-api.version}</version>
```
Comment (jimma) [inline ee-9/source-transform/webservices/server-integration/pom.xml:71]: We should use this artifact: 
```
<dependency>
    <groupId>org.jboss.spec.jakarta.xml.ws</groupId>
    <artifactId>jboss-jakarta-xml-ws-api_3.0_spec</artifactId>
    <version>1.0.0.Final</version>
</dependency>
```
And add this to parent pom dependency management too.

File: ee-9/source-transform/webservices/server-integration/pom.xml [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+       <artifactId>jbossws-common</artifactId>
+       <version>${jbossws.common.version}</version> <!-- todo remove version as well as exclude -->
+       <exclusions>
+          <exclusion>
+             <groupId>org.jboss.ws</groupId>
+             <artifactId>jbossws-spi</artifactId>
+          </exclusion>
+       </exclusions>
+    </dependency>
+
```
Comment (jimma) [inline ee-9/source-transform/webservices/server-integration/pom.xml:190]: We need to add this dependency to dependency management .

**Comment by rsearls** [comment]
This PR has a dependency on https://issues.redhat.com/browse/WFLY-15549 and can not proceed without it.

File: ee-9/source-transform/webservices/server-integration/pom.xml [hunk 3]
```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2010, Red Hat, Inc., and individual contributors
```
Comment (rhusar) [inline ee-9/source-transform/webservices/server-integration/pom.xml:4]: s/2010/2021 :-)

**Pull request closed without merging**